clone.py

```diff
@@ -165,8 +165,11 @@ def _make_filename(self, url):
                     data = str(soup).encode()
                 with open(os.path.join(self.target_path, hash_name), 'wb') as index_fh:
                     index_fh.write(data)
-                if content_type == 'text/css':
-                    css = cssutils.parseString(data)
+                if content_type == 'text/css':                       
+                    if self.css_validate == "False":
+                        css = cssutils.parseString(data, validate=False)
```

Comment:
Maybe it's better to write:
```cssutils.parseString(data, validate=self.validate)```
In this case you don't need this if/else statement